src/shell/command_helper.h

```diff
@@ -36,6 +36,7 @@
 #include <dsn/perf_counter/perf_counter_utils.h>
 #include <dsn/utility/string_view.h>
 #include <dsn/utils/time_utils.h>
```

Comment:
Could you add some performance tests for this pull request?

Reply:
done